cmd/convox/install.go

```diff
@@ -111,6 +111,10 @@ func init() {
 				Usage: "internet gateway id to use in existing vpc",
 			},
 			cli.BoolFlag{
+				Name:  "no-autoscale",
```

Comment:
This could be a `BoolTFlag` named `autoscale` and then both `--autoscale` and `--no-autoscale` should work with true being the default.